draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -190,7 +190,7 @@
    (<xref target="RFC7230"/> through <xref target="RFC7235"/>).
 </t>
 <t>
-   HTTP/2 (<xref target="HTTP2"/>) introduced a multiplexed session layer
+   HTTP/2 (<xref target="RFC7540"/>, <xref target="HTTP2"/>) introduced a multiplexed session layer
```

Comment:
I don't think that is important here, since the reference is on the name; it should cite the current spec-to-be. If we changed the paragraph so that it reads more like the one above, which talks first about the protocol and then the history of RFCs that defined it, then I would cite the actual revisions.

Reply:
ack